src/backend/vulkan/src/pool.rs

```diff
@@ -71,12 +71,19 @@ impl pool::RawCommandPool<Backend> for RawCommandPool {
         self.device.0.free_command_buffers(self.pool, &buffers);
     }
 
-    unsafe fn from_queue(queue: &CommandQueue) -> RawCommandPool {
-        // Create command pool
+    unsafe fn from_queue(queue: &CommandQueue, create_flags: pool::CommandPoolCreateFlags) -> RawCommandPool {
+        let mut flags = vk::CommandPoolCreateFlags::empty();
+        if create_flags.contains(pool::TRANSIENT) {
+            flags |= vk::COMMAND_POOL_CREATE_TRANSIENT_BIT;
+        }
+        if create_flags.contains(pool::RESET_INDIVIDUAL) {
```

Comment:
I believe there is a semantic mistake here.
`vk::COMMAND_POOL_CREATE_RESET_COMMAND_BUFFER_BIT` says that a command buffer is reset automatically when `begin` is called (and resources freed).
So it's not defining the capability to reset command buffers, like `pool::RESET_INDIVIDUAL` suggests. AFAIS, you can always reset individual `vkCommandBuffer` things.

Reply:
According to https://www.khronos.org/registry/vulkan/specs/1.0/man/html/vkResetCommandBuffer.html it's not possible to call `vkCmdResetCommandBuffer` without settings this flag.

Reply:
I wasn't aware of the implicit reset behavior of `BeginCommandBuffer` in combination of this flag. This is something I need to address for the GL backend as it's currently not done.